<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>AppointmentEditView</class>
<widget class="QWidget" name="AppointmentEditView">
 <property name="geometry">
  <rect>
   <x>0</x>
   <y>0</y>
   <width>600</width>
   <height>500</height>
  </rect>
 </property>
 <property name="windowTitle">
  <string>预约编辑</string>
 </property>
 <layout class="QVBoxLayout" name="verticalLayout">
  <item>
   <widget class="QGroupBox" name="groupBox">
    <property name="title">
     <string>预约信息</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <!-- 预约ID（只读） -->
     <item row="0" column="0">
      <widget class="QLabel" name="label_id">
       <property name="text">
        <string>预约ID：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="txtAppointmentId">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>自动生成</string>
       </property>
      </widget>
     </item>
     
     <!-- 患者ID输入 -->
     <item row="1" column="0">
      <widget class="QLabel" name="label_patient">
       <property name="text">
        <string>患者ID：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_patient">
       <item>
        <widget class="QLineEdit" name="txtPatientId">
         <property name="placeholderText">
          <string>请输入患者ID（可留空）</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSearchPatient">
         <property name="text">
          <string>查询患者</string>
         </property>
         <property name="maximumWidth">
          <number>80</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     
     <!-- 医生ID输入 -->
     <item row="2" column="0">
      <widget class="QLabel" name="label_doctor">
       <property name="text">
        <string>医生ID：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_doctor">
       <item>
        <widget class="QLineEdit" name="txtDoctorId">
         <property name="placeholderText">
          <string>请输入医生ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSearchDoctor">
         <property name="text">
          <string>查询医生</string>
         </property>
         <property name="maximumWidth">
          <number>80</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     
     <!-- 患者姓名和医生姓名显示（只读） -->
     <item row="3" column="0">
      <widget class="QLabel" name="label_patientName">
       <property name="text">
        <string>患者姓名：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtPatientName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>输入患者ID后显示</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_doctorName">
       <property name="text">
        <string>医生姓名：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="txtDoctorName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>输入医生ID后显示</string>
       </property>
      </widget>
     </item>
     
     <!-- 预约日期和时间 -->
     <item row="4" column="0">
      <widget class="QLabel" name="label_date">
       <property name="text">
        <string>预约日期：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="dateAppointmentDate">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="minimumDate">
        <date>
         <year>2024</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="maximumDate">
        <date>
         <year>2025</year>
         <month>12</month>
         <day>31</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_time">
       <property name="text">
        <string>时间：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QTimeEdit" name="timeAppointmentTime">
       <property name="displayFormat">
        <string>HH:mm</string>
       </property>
      </widget>
     </item>
     
     <!-- 预约事由 -->
     <item row="5" column="0">
      <widget class="QLabel" name="label_reason">
       <property name="text">
        <string>预约事由：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QLineEdit" name="txtReason">
       <property name="placeholderText">
        <string>请输入预约事由</string>
       </property>
      </widget>
     </item>
     
     <!-- 症状描述 -->
     <item row="6" column="0">
      <widget class="QLabel" name="label_symptoms">
       <property name="text">
        <string>症状描述：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="3">
      <widget class="QPlainTextEdit" name="txtSymptoms">
       <property name="maximumHeight">
        <number>60</number>
       </property>
       <property name="placeholderText">
        <string>请输入症状描述（可选）</string>
       </property>
      </widget>
     </item>
     
     <!-- 优先级 -->
     <item row="7" column="0">
      <widget class="QLabel" name="label_priority">
       <property name="text">
        <string>优先级：</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="cmbPriority">
      </widget>
     </item>
     
     <!-- 状态 -->
     <item row="7" column="2">
      <widget class="QLabel" name="label_status">
       <property name="text">
        <string>状态：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QComboBox" name="cmbStatus"/>
     </item>
     
     <!-- 备注 -->
     <item row="8" column="0">
      <widget class="QLabel" name="label_notes">
       <property name="text">
        <string>备注：</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="3">
      <widget class="QPlainTextEdit" name="txtNotes">
       <property name="maximumHeight">
        <number>60</number>
       </property>
       <property name="placeholderText">
        <string>请输入备注信息（可选）...</string>
       </property>
      </widget>
     </item>
     
     <!-- 时间冲突警告 -->
     <item row="9" column="0" colspan="4">
      <widget class="QLabel" name="lblTimeConflict">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </item>
  
  <!-- 错误信息 -->
  <item>
   <widget class="QLabel" name="lblError">
    <property name="text">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </item>
  
  <!-- 按钮区域 -->
  <item>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </item>
 </layout>
</widget>
<resources/>
<connections/>
</ui>